registration-default.properties

```diff
@@ -472,18 +472,12 @@ mosip.kernel.virus-scanner.host=${mosip.kernel.virus-scanner.host}
 mosip.kernel.virus-scanner.port=${mosip.kernel.virus-scanner.port}
 mosip.kernel.otp.expiry-time=${mosip.kernel.otp.expiry-time}
 ## end of global properties
-
-# Support Reg.Client upgrade
-mosip.registration.verion.upgrade.version-mappings={ "1.1.4":{ "dbVersion":"1.1.4", "releaseOrder":1}, "1.1.5":{ "dbVersion":"1.1.5", "releaseOrder":2}, "1.1.5.5":{ "dbVersion":"1.1.5.5", "releaseOrder":3}, "1.2.0.1-SNAPSHOT":{ "dbVersion":"1.2.0.1", "releaseOrder":4} }
-
-## backward compatibility fields	
-mosip.registration.uin.update.configured.fields=test	
-mosip.primary-language=eng
-mosip.secondary-language=ara
+## backward compatibility fields
+mosip.registration.uin.update.configured.fields=test
+mosip.primary-language=${mosip.primary-language}
+mosip.secondary-language=${mosip.secondary-language}
 mosip.registration.ideal_time=900	
-mosip.kernel.applicant.type.age.limit=${mosip.kernel.applicant.type.age.limit}
-mosip.registration.max_age=200
```

Comment:
We should add these properties back.

Reply:
its already there removed the duplicate one